### DISCUSSION TITLE
Russian translate

### DISCUSSION THREAD

**Discussion by Rejinu** [Lurkars/gloomhavensecretariat#624] [Ideas]
Hey. My name is Timur. I have russian version of base Gloomhaven, FC and going to buy Crimson Scale and Jaws of the Lion. Is there is a way for me to "translate" this parts of your app on a voluntary basis?

**Comment by Lurkars** [comment]
Hey, basic translation is under following files:

https://github.com/Lurkars/gloomhavensecretariat/tree/main/src/assets/locales

So just pick English file and create a corresponding ru.json file.
For edition data (like Names, Ability texts etc.) there are folder in the edition data. For example for Gloomhaven see:

https://github.com/Lurkars/gloomhavensecretariat/tree/main/data/gh/label and

https://github.com/Lurkars/gloomhavensecretariat/tree/main/data/gh/label-spoiler

Then just file a PR with the translation.
